package-lock.json

```diff
@@ -9,9 +9,9 @@
       "resolved": "https://registry.npmjs.org/@jaredpalmer/after/-/after-1.3.1.tgz",
```

Comment:
Please the changes to the package-lock, there is no reason for it to change in this PR.